python/prairielearn_test.py

```diff
@@ -1,15 +1,138 @@
 # lol
 import sys
 import os
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../lib')))
 
+sys.path.insert(
```

Comment:
@nwalters512 Does this line need to be here? When I removed it I had no issues running pytest.